queries.sql

```diff
@@ -23,3 +23,51 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 /*Animals weighing between 10.4kg and 17.3kg*/
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- To update the animals table inside a transaction and set the species column to unspecified
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals WHERE species = 'unspecified';
+ROLLBACK TRANSACTION;
```

Comment:
- [ ] 👋 Hello! Fantastic job in crafting a transaction. 🛠️ Initially, you created a transaction with the intention of updating a species to "unspecified." 🐾 However, after encountering a situation that required a rollback, you skillfully reversed those changes. 🔄 Now, it's crucial to ensure that the modifications have indeed been completely undone. 🕵️‍♂️ Let's double-check to confirm that everything is back to its original state. 🧐🔍


```suggestion
SELECT species from animals;
```